ons_alpha/jinja2/templates/pages/bulletins/views.py

```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
```

Comment:
This view is in the templates directory, but urls.py uses `from ons_alpha.bulletins.views import latest_bulletin_view, previous_releases_view`

which causes Django to fail to start all together